homeassistant/components/solaredge/__init__.py

```diff
@@ -1,20 +1,26 @@
 """The solaredge component."""
+import asyncio
+
 import voluptuous as vol
 
 from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
-from homeassistant.const import CONF_API_KEY, CONF_NAME
+from homeassistant.const import CONF_API_KEY, CONF_NAME, CONF_PASSWORD, CONF_USERNAME
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.typing import HomeAssistantType
 
 from .const import CONF_SITE_ID, DEFAULT_NAME, DOMAIN
 
+PLATFORMS = ["sensor", "light"]
+
 CONFIG_SCHEMA = vol.Schema(
     {
         DOMAIN: vol.Schema(
             {
                 vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
                 vol.Required(CONF_API_KEY): cv.string,
                 vol.Required(CONF_SITE_ID): cv.string,
+                vol.Optional(CONF_USERNAME): cv.string,
+                vol.Optional(CONF_PASSWORD): cv.string,
             }
```

Comment:
Why do we need the username and password if we already have an api key?

Reply:
It is a different service that is not accessible with the api key.